crates/weaver_resolver/src/any_value.rs

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Functions to resolve a semantic convention body.
+
+use weaver_resolved_schema::any_value::AnyValue;
+use weaver_semconv::{any_value::AnyValueSpec, attribute::EnumEntriesSpec};
+
+/// Resolve a `Body` specification into a resolved `Body`.
+#[must_use]
+pub fn resolve_any_value_spec(value: &AnyValueSpec) -> AnyValue {
```

Comment:
This resolution looks like a deep copy. See my previous comments, I don't think we need this code.

Reply:
I'll update, as I was just following the existing pattern (and it was previously needed for the body), but I can rework -- the only possible minor issue will be the "r#type" field of the AnyValue which doesn't explicitly exist (as a separate field -- as it's derived from the enum structure and the `Display`), but I'm sure there is some way around that.

Reply:
And yes, this is effective just a deep conversion / copy as it is now.